### DISCUSSION TITLE
GITHUB EDUCATION  APPLICATION APPROVED BUT BENEFITS NOT GRANTED TO MY ACCOUNT

### DISCUSSION THREAD

**Discussion by aravind0543** [community/community#132023] [GitHub Education]
### Hi everyone,

Hello I was verified with my academic details and was approved but still not get the benefits of student developer pack its been more than 2 days.
<img width="321" alt="Screenshot 2024-07-13 073317" src="https://github.com/user-attachments/assets/5cf5354a-13e8-47df-a6bf-e7414c18cb4f">

**Comment by naveenalla3000** [comment]
@akuthotaaravind wait for 2-4 days after approval, you will receive.
mark me answer if it helpful☺️.

**Comment by davevad93** [comment]
Hi @akuthotaaravind ,

The message says that your benefits will be available soon, when available the status label will turn purple and change to:

`Congrats! Your request for @akuthotaaravind was approved and your benefits are now available`

That could take a while, usually it's a matter of a few days but sometimes it can take even a couple of weeks, the waiting time may vary depending on different factors e.g. the volume of applications.
